book/src/developers/contributing/releases/release_checklist.md

```diff
@@ -15,17 +15,36 @@ anything.
 
 Make sure that version follows [semver](https://semver.org/) rules e.g (`0.2.0-alpha.3`).
 
-**For the time being, ALWAYS specify the `-alpha` suffix.**
+Since trin is now stable, but v0, breaking changes are a minor version bump,
+and all other changes are a patch version bump. Updating to v1 would require a
+group discussion and decision.
 
-## Release the version
+## Release dependencies
+
+For now, that's just ethportal-api. Manually bump the version in the
+Cargo.toml, and run `cargo update` to update the workspace lockfile. Commit and
+merge these changes to trin. Then run:
+
+```bash
+cd ethportal-api
+cargo publish --no-verify
+```
+
+We would like to get rid of the no-verify ASAP, but for now Cargo.lock is
+causing issues that we have no other workaround for. `cargo publish` generates
+a new lock file, and then complains that the lockfile is new. See this
+[StackOverflow
+post](https://stackoverflow.com/questions/79276315/how-to-build-a-cargo-lock-file-for-a-package-within-a-workspace).
```

Comment:
```suggestion
[StackOverflow post](https://stackoverflow.com/questions/79276315/how-to-build-a-cargo-lock-file-for-a-package-within-a-workspace).
```

It is a little weird that these are on 2 different lines

Reply:
yeah, vim just did that with gqq, but I don't mind manually changing it